core/camel-management/pom.xml

```diff
@@ -85,6 +85,13 @@
             <artifactId>log4j-slf4j-impl</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
We do NOT want this dependency

Reply:
I need this dependency to test if the result is Serializable. I understand your point about debugger being used by many different tools, perhaps we need another method which evaluates an expression and converts it to String?

Reply:
No problem, I actually figured out how to check if the object is serializable without using third party library, it's gone now.